arwen/elrondapi/bigIntOps.go

```diff
@@ -278,13 +286,16 @@ func bigIntGetUnsignedArgument(context unsafe.Pointer, id int32, destination int
 		return
 	}
 
-	value := bigInt.GetOne(destination)
+	value, err := bigInt.GetOne(destinationHandle)
```

Comment:
for this case you need GetOrCreate

Reply:
same for here. "value" is created before the function call in the rust framework